Dockerfile

```diff
@@ -312,6 +312,7 @@ RUN if [[ "$TARGETPLATFORM" == linux/arm64 ]]; then \
       ; \
     fi
 
+RUN pip3 install pathogen-embed==2.0.0
```

Comment:
_(non-blocking)_

Noting from the [run summary](https://github.com/nextstrain/docker-base/actions/runs/9603060473) that this added 8.5 minutes to the build time:

```
[linux/arm64] RUN pip3 install pathogen-embed==2.0.0     | 496.7s (61.9%)  ████████████████████
[linux/amd64] RUN pip3 install pathogen-embed==2.0.0     | 17.0s (2.1%)    ▋
```

I'm not worried since this dependency is pinned, meaning the cached result will be used most of the time and that the increased build time would only be noticeable during (infrequent) cache misses.

Reply:
This one change also adds ~57 MB to the image file size... 😕 

I expected an increase, since we need to install scikit-learn, HDBSCAN, and UMAP, but it is kind of a bummer.